### ISSUE TITLE
Redundant Site Pages and Re-organisation

### ISSUE THREAD

**Issue by agardnerIT** [keptn/keptn.github.io#1345]
This thread will primarily focus as a discussion record with other issues being spawned as we come to some decisions.

# Status Quo
Keptn website has the following pages (only listing those pertinent to this thread):

- Why Keptn? (Keptn overview of key use cases)
- Keptn and Other tools (How does Keptn compare / work with other tooling)
- Explore Keptn (Link to zero-install Keptn playground + next steps and installation instructions)
- Quick Start (outdated installation methods and tutorials)
- Installation (more installation instructions)

# My Opinion
As a new Keptn user (first time on the site), I have 3 goals:

1. What is Keptn and why should I care about it?
2. How do I play around quickly with Keptn (show me the goods + let me play with the goods)
3. I like the goods. How do I install?

As a returning Keptn user (I'm "sold" on Keptn and potentially using it). I'm here for:

1. I need some Keptn concepts to mentally visualise how to set things up)
2. Reference material (I need to know how to do something eg. a particular feature flag or helm chart --set value)

# The Current Issues (My Opinion)

## Why Keptn Page
"Why Keptn" page is great but it should like at the end to the "keptn and other tools page". When I'm finished reading the "why keptn" page I'm probably thinking of all the other tools I know about that sound like they do a similar thing. Explain to the reader that Keptn isn't a replacement for those tools but a complementary orchestrator that adds DevOps and SRE capabilities out-of-the-box.

## Keptn and Other Tools Page
"Keptn and other tools" this page was created because, in my experience, it's the most common confusion point for new adopters. We should direct people here more from wherever is appropriate.

## Explore Keptn Page

"Explore Keptn". This flow of this page is great. It links me to the zero-install playground (get me started quick) and then (if / when I've decided I like Keptn) it should me how to easily install with an industry standard tool (helm). It links me to the advanced helm docs if I need them. Finally it links me to Slack if I'm stuck. This page is all I need as a new user who is ready to Install Keptn

## Quick Start Page

This is a legacy page and is now redundant. It starts with a helm install command (duplicating the explore keptn page). It shows me only 2 execution services (where it feels like the community is moving towards the job executor service + web hook service). Indeed the jmeter and helm services are soon to be removed from the core bundle.

Then I get `thekindkeptn` instructions which is outdated and running Keptn version 0.16.0. The page then suggests I try multistage delivery and auto-remediation tutorials. Neither of which work with modern Keptn versions. `thekindkeptn` being out of date is deliberate as I (as the maintainer) want to push people to the easier zero-install option via browser.

Then there are k3d instructions which add zero value alongside thekindkeptn as that's exactly what `thekindkeptn` uses "behind the scenes".

Then I see installation instructions for k3d - why the Keptn website is showing me how to install k3d I don't know - just link to their docs instead.

Then I see the now unsupported `keptn install` command which no longer works.

Then I see the multi-stage delivery and auto-healing tutorials (which, as above, don't work).

Then I see an include of the `tutorials.keptn.sh` all of which are terribly out of date and most are unmaintained.

All in all, the Quickstart page should be deleted ASAP.

## Installation
This page shows LOTS of information and links to other sections. I see Istio listed which **used** to be mandatory but is another common confusion point for new Keptn users (and a potential adoption blocker as they don't want / can't install Istio).

Then the instructions try to tell me the entire world.

In short, I'm struggling to see what value this page offers over the "explore keptn" page when the explore page is used in conjunction with the "advanced" or "deep" technical pages.
